include/seastar/parquet/bpacking.hh

```diff
@@ -0,0 +1,3819 @@
+#pragma once
```

Comment:
I wonder if it passes our review :) It kind of looks like something that compiler would optimize anyway. But since it's an external, machine-generated file, let it stay for now. It should probably have a copy of its original license somewhere on top.

Reply:
Added a licence notice in e3555d1.